app/controllers/applicants_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ApplicantsController < ApplicationController
+  def show
+    @applicant = Applicant.find(params[:id])
+    @applicant_pets = @applicant.pets
+    if params[:pet_name].present?
+      @pets = Pet.search(params[:pet_name])
+    else
+      @pets = []
+    end
+    @submission_ready = @applicant.descript
+    @not_submitted = @applicant.no_description?
+  end
+
+   def index
+    @applicants = Applicant.all
+  end
+
+  def new
+  end
+
+  def create
+    @applicant = Applicant.new(applicant_params)
+    if @applicant.save
+      redirect_to "/applicants/#{Applicant.last.id}"
```

Comment:
here I think you could probably do 
```ruby
redirect_to "/applicants/#{@applicant.id}"
```
where you are grabbing the `id` that is created with your `.new` on line 22.